#! /bin/sh /usr/share/dpatch/dpatch-run
## debian_dont_link_libmono.dpatch by Jo Shields <directhex@apebox.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Force cflags not to link against libmono0 needlessly

@DPATCH@
diff -urNad f-spot-0.5.0.3~/configure.in f-spot-0.5.0.3/configure.in
--- f-spot-0.5.0.3~/configure.in	2009-07-08 23:33:57.000000000 +0100
+++ f-spot-0.5.0.3/configure.in	2009-07-08 23:34:17.000000000 +0100
@@ -69,7 +69,7 @@
 LCMS_REQUIRED=1.12
 dnl -- LIBGPHOTO2_REQUIRED=2.1.4
 
-PKG_CHECK_MODULES(F, libgnome-2.0 >= $LIBGNOME_REQUIRED libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED libexif >= $LIBEXIF_REQUIRED_MIN libexif < $LIBEXIF_REQUIRED_MAX glade-sharp-2.0 >= $GTKSHARP_REQUIRED gnome-vfs-sharp-2.0 >= $GTKSHARP_REQUIRED gtk+-2.0 >= $GTK_REQUIRED mono >= $MONO_REQUIRED)
+PKG_CHECK_MODULES(F, libgnome-2.0 >= $LIBGNOME_REQUIRED libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED libexif >= $LIBEXIF_REQUIRED_MIN libexif < $LIBEXIF_REQUIRED_MAX glade-sharp-2.0 >= $GTKSHARP_REQUIRED gnome-vfs-sharp-2.0 >= $GTKSHARP_REQUIRED gtk+-2.0 >= $GTK_REQUIRED)
 F_LIBS="$F_LIBS -r:Mono.Cairo"
 AC_SUBST(F_CFLAGS)
 AC_SUBST(F_LIBS)
